validations/kyc.test.ts

```diff
@@ -58,6 +64,52 @@ describe('/kyc', () => {
       Object.values(KycStatus),
     )
   })
+  if (config.clientApiKey && config.providerId) {
+    it('sends kyc webhooks', async () => {
+      const beforeKycAdd = new Date().toISOString()
+      const wallet = ethers.Wallet.createRandom()
+      const fiatConnectClient = new FiatConnectClient(
+        {
+          baseUrl: config.baseUrl,
+          network: Network.Alfajores,
+          accountAddress: wallet.address,
+          apiKey: config.clientApiKey,
+        },
+        (message: string) => wallet.signMessage(message),
+      )
+      const loginResult = await fiatConnectClient.login()
+      expect(loginResult.isOk).to.be.ok
+
+      const addKycResult = await fiatConnectClient.addKyc(mockKYCInfo)
+      expect(addKycResult.isOk).to.be.true
+
+      const getKycResult = await fiatConnectClient.getKycStatus({
+        kycSchema: mockKYCInfo.kycSchemaName,
+      })
+      expect(getKycResult.isOk).to.be.true
+      expect(getKycResult.unwrap().kycStatus).to.be.oneOf(
+        Object.values(KycStatus),
+      )
+      await new Promise((resolve) => setTimeout(resolve, 3000)) // Wait a bit for API requests to process
```

Comment:
this will slow down tests and may not always be necessary. It also bakes in a race condition. Instead, it might be preferable to do some kind of retries with backoff.